apps/src/StudioApp.js

```diff
@@ -1,5 +1,6 @@
 /* global Blockly, ace:true, $, requirejs, marked */
 
+var aceMode = require('./acemode/mode-javascript_codeorg');
```

Comment:
Does this do okay in other apps where we haven't required ace (i.e. during initApp.js)?